lib/node_modules/@stdlib/math/base/special/cidentity/manifest.json

```diff
@@ -24,15 +26,51 @@
 	],
 	"confs": [
 		{
-			"src": [
-				"./src/main.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [],
-			"libpath": [],
-			"dependencies": []
+		  "task": "build",
+		  "src": [
+			"./src/main.c"
+		  ],
+		  "include": [
+			"./include"
+		  ],
+		  "libraries": [],
+		  "libpath": [],
+		  "dependencies": [
+			"@stdlib/math/base/napi/unary",
+			"@stdlib/complex/float64",
+			"@stdlib/complex/reim"
+		  ]
+		},
+		{
+		  "task": "benchmark",
+		  "src": [
+			"./src/main.c"
+		  ],
+		  "include": [
+			"./include"
+		  ],
```

Comment:
This file has mixed TAB/SPACE indentation. Provided EditorConfig is setup, this should be automatically handled.